src/client/app/+global-settings/+cloth-settings/cloth-settings.component.ts

```diff
@@ -0,0 +1,12 @@
+import { Component } from '@angular/core';
+
+/**
+ * This class represents the lazy loaded ClothSettingsComponent.
+ */
+@Component({
+  moduleId: module.id,
+  selector: 'ap-cloth-settings',
+  templateUrl: 'cloth-settings.component.html',
+  styleUrls: ['cloth-settings.component.css']
```

Comment:
please remove this , get the information from karthik , which file u need to include for styles